charts/pg-operator/values.yaml

```
@@ -0,0 +1,140 @@
+---
+# ======================
+# Installer Controls
+# ======================
+fullnameOverride: ""
+
+# rbac: settings for deployer RBAC creation
+rbac:
+    # rbac.create: if false RBAC resources should be in place
+    create: true
+    # rbac.useClusterAdmin: creates a ClusterRoleBinding giving cluster-admin to serviceAccount.name
+    useClusterAdmin: false
+
+# serviceAccount: settings for Service Account used by the deployer
+serviceAccount:
+    # serviceAccount.create: Whether to create a Service Account or not
+    create: true
+    # serviceAccount.name: The name of the Service Account to create or use
+    name: ""
+
+archive_mode: "true"
+archive_timeout: "60"
+backrest_aws_s3_bucket: ""
+backrest_aws_s3_endpoint: ""
+backrest_aws_s3_key: ""
+backrest_aws_s3_region: ""
+backrest_aws_s3_secret: ""
+backrest_aws_s3_uri_style: ""
+backrest_aws_s3_verify_tls: "true"
+backrest_gcs_bucket: ""
+backrest_gcs_endpoint: ""
+backrest_gcs_key_type: ""
+backrest_port: "2022"
+badger: "false"
+ccp_image_prefix: "percona/percona-postgresql-operator"
+ccp_image_pull_secret: ""
+ccp_image_pull_secret_manifest: ""
+ccp_image_tag: "0.1.0-postgres-ha"
+create_rbac: "true"
+crunchy_debug: "false"
+db_name: ""
+db_password_age_days: "0"
+db_password_length: "24"
+db_port: "5432"
+db_replicas: "0"
+db_user: "testuser"
+default_instance_memory: "128Mi"
+default_pgbackrest_memory: "48Mi"
+default_pgbouncer_memory: "24Mi"
+default_exporter_memory: "24Mi"
+delete_operator_namespace: "false"
+delete_watched_namespaces: "false"
+disable_auto_failover: "false"
+disable_fsgroup: "false"
+reconcile_rbac: "true"
+exporterport: "9187"
+metrics: "false"
+namespace: "pgo"
+namespace_mode: "dynamic"
+pgadmin_storage: "default"
+pgbadgerport: "10000"
+pgo_add_os_ca_store: "false"
+pgo_admin_password: "examplepassword"
+pgo_admin_perms: "*"
+pgo_admin_role_name: "pgoadmin"
+pgo_admin_username: "admin"
+pgo_apiserver_port: "8443"
+pgo_apiserver_url: "https://postgres-operator"
+pgo_client_cert_secret: "pgo.tls"
+pgo_client_container_install: "false"
+pgo_client_install: "true"
+pgo_client_version: "4.7.0"
```

```suggestion
pgo_client_version: "4.6.2"
```
or maybe we can remove it at all

---

done